test/ng/directive/inputSpec.js

```diff
@@ -1263,7 +1263,7 @@ describe('input', function() {
         $rootScope.halfSecondToNextYear = new Date(2013, 11, 31, 23, 59, 59, 500);
       });
 
-      expect(inputElm.val()).toBe('2013-12-31T23:59:59.500');
+      expect(inputElm.val()).toMatch(/^2013-12-31T23:59:59.50*$/);
```

Comment:
@urikalish please explain why this is needed, how does value look like really in this case, why would we need to switch to regex

Reply:
The value I get is '2013-12-31T23:59:59.5'. In all these cases, we get "truncated" numbers.